src/flake8_to_ruff/tool_configs.rs

```diff
@@ -0,0 +1,17 @@
+use std::path::Path;
+
+use anyhow::Result;
+
+use super::{black::Black, isort::Isort, parse_black_options, parse_isort_options};
+
+pub struct ToolConfigs {
+    pub black: Option<Black>,
+    pub isort: Option<Isort>,
```

Comment:
Does it matter that the struct takes ownership of `Black` and `Isort` respectively? Ideally these could be references but I had a hard time making that play nice in `parse_tool_configs` below 👇🏼

Reply:
Not a big deal. You could probably introduce a lifetime to do something like:

```rs
pub struct ToolConfigs<'a> {
    pub black: Option<&'a Black>,
    pub isort: Option<&'a Isort>,
}
```

But then you'd have to ensure that the values outlive this struct. So you'd need to parse the values at a level above `parse_tool_configs` (like, in `main.rs`), then pass them to `ToolConfigs` as references.

Reply:
Added lifetime in https://github.com/charliermarsh/ruff/pull/2082/commits/8c8c497b97b7eb87a8afe67d9d3876ae1559f6d9. Happy to revert that commit if the previous approach is preferable too 👍🏼